webui/react/src/pages/ExperimentDetails/ExperimentTrials.tsx

```diff
@@ -101,6 +101,13 @@ const ExperimentTrials: React.FC<Props> = ({ experiment }: Props) => {
       </Link>
     );
 
+    const restartsRenderer = (_: string, record: TrialItem): React.ReactNode => {
```

Comment:
suggestion. only render if there has been a restart? these should be rare cases.

- also if it only shows up conditionally it'd be more eye-grabbing which is probably a reason behind why they asked for the feature.

Reply:
Agreed. This makes a lot of sense.